scripts/test-infra/mdlint.sh

```diff
@@ -1,7 +1,19 @@
 #!/bin/bash -e
 
-# Install mdl
-gem install mdl -v 0.11.0
+IS_CONTAINER=${IS_CONTAINER:-false}
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-docker}"
 
-# Run verify steps
-make mdlint
+if [ "${IS_CONTAINER}" != "false" ]; then
+    # Install mdl
+    gem install mdl -v 0.11.0
+    # Run verify steps
+    find docs/ -path docs/vendor -prune -false -o -name '*.md' -exec mdl -s docs/mdl-style.rb {} \+
+else
+  "${CONTAINER_RUNTIME}" run --rm \
+    --env IS_CONTAINER=TRUE \
+    --volume "${PWD}:/workdir:ro,z" \
+    --entrypoint sh \
+    --workdir /workdir \
+    ruby:slim \
+    /workdir/scripts/test-infra/mdlint.sh "${@}"
+fi;
```

Comment:
Somehow this feels too complicated. Should be super simple. The "philosophy" has been that `scripts/test-infra` are as simple as possible and basically only supposed to be called from prox.

```suggestion
# Install mdl
gem install mdl -v 0.11.0

find docs/ -path docs/vendor -prune -false -o -name '*.md' | xargs $(MDL) -s docs/mdl-style.rb
```